docs/calamari/SDK.md

```diff
@@ -0,0 +1,292 @@
+- `errorCallback`, callback function after an error has occured, set by default to `null`.
+
+## Transacting
+
+After initialization of the `MantaPrivateWallet` class, `initialWalletSync()` must be called before any transactions are made.
+
+After every single transaction, to get the latest data from the ledger, `walletSync()` must be called.
+
+A PolkadotJS `Signer` and public PolkadotJS `Address` should be provided to every function that requires transacting. Below is an example of how to get these values, this example assumes that the Polkadot JS extension is installed and contains an existing account.
+
+### Polkadot JS Transaction Parameters
+
+```javascript
+import { web3Accounts, web3Enable, web3FromSource } from '@polkadot/extension-dapp';
+
+// Get Polkadot JS Signer and Polkadot JS account address.
+const getPolkadotSignerAndAddress = async () => {
+    const extensions = await web3Enable('Polkadot App');
+    if (extensions.length === 0) {
+        throw new Error("Polkadot browser extension missing. https://polkadot.js.org/extension/");
+    }
+    const allAccounts = await web3Accounts();
+    let account = allAccounts[0];
+
+    const injector = await web3FromSource(account.meta.source);
+    const polkadotSigner = injector.signer;
+    const polkadotAddress = account.address;
+    return {
+        polkadotSigner,
+        polkadotAddress
+    }
+}
+```
+
+Below is an example of how to transact using fungible tokens, there are four main methods that `manta-pay` provides:
+- `toPrivateSend(asset, amount, polkadotSigner, polkadotAddress)`
+- `privateTransferSend(asset, amount, receiver, polkadotSigner, polkadotAddress)`
+- `toPublicSend(asset, amount, polkadotSigner, polkadotAddress)`
+- `publicTransfer(asset, amount, destinationAddress, polkadotSigner, polkadotAddress)`
+
+> This example assumes the `polkadotAddress` already has associated public funds.
+
+### To Private
+
+This example converts 10 public DOL tokens to 10 private DOL tokens.
+
+```javascript
+// DOL token
+const assetId = new BN("1");
+const amount = new BN("10000000000000000000");
+
+// Sync with most recent ledger state.
+await privateWallet.initialWalletSync();
+
+// Get private address
+const privateAddress = await privateWallet.getPrivateAddress();
+
+// Get private balance of DOL for given private address
+const privateBalance = await privateWallet.getPrivateBalance(assetId);
+
+// Privatize 10 DOL to 10 pDOL
+await privateWallet.toPrivateSend(assetId, amount, polkadotSigner, polkadotAddress);
+
+// Sync to get latest data after the transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should be incremented by 10 units.
+const newPrivateBalance = await mantaSdk.getPrivateBalance(assetId);
+```
+
+### Private Transfer
+
+This example transfers 10 private private pDOL to another address.
+
+```javascript
+// DOL token
```

Comment:
In this doc, `DOL` or `pDOL` should be `KMA` or `pKMA`.

Reply:
Should be `zkKMA`.

Reply:
done